docker/Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN echo 'set visualbell' >> /root/.vimrc
 # Add DOI CA to local CAs so that SSL can work over VPN
 COPY DOIRootCA2.crt /usr/local/share/ca-certificates
 RUN update-ca-certificates
+ENV CURL_CA_BUNDLE /etc/ssl/certs/ca-certificates.crt
```

Comment:
Fixes an annoying python bug which probably isn't relevant to this project - just here for consistency in case somebody copies this file.